src/widgets/WidgetMainHummGroup.vue

```diff
@@ -73,6 +73,7 @@ export default defineComponent({
     return {
       isWidgetOpen: true,
       isDialogOpen: false,
+      displayedProduct: this.products[0],
       buttonCloseLabel: 'Close',
       Theme: ThemeEnum,
     }
```

Comment:
This widget is not part of MVP so I'm just maintaining them to the point of not breaking them.